components/compliance-service/ingest/ingestic/mappings/comp-run-info.go

```diff
@@ -8,24 +8,163 @@ var ComplianceRunInfo = Mapping{
   {
     "index_patterns": ["` + IndexNameComplianceRunInfo + `"],
     "settings": {
-      "index": {
-        "refresh_interval": "1s",
-        "number_of_shards": "5"
-      }
+        "index": {
+            "refresh_interval": "1s",
+            "number_of_shards": "5"
+        },
+        "analysis": {
```

Comment:
what is the need for these analyzers/tokenizer here? Do we need this for filtering based on a field?

Reply:
Yes @Dmaddu , We need this analyzers/tokenizer for filtering based on various fields.